apps/emqx_authz/src/emqx_authz.erl

```diff
@@ -155,8 +157,8 @@ do_update({?CMD_APPEND, Sources}, Conf) when is_list(Sources), is_list(Conf) ->
     NConf = Conf ++ Sources,
     ok = check_dup_types(NConf),
     NConf;
-do_update({{?CMD_REPLACE, Type}, #{<<"enable">> := true} = Source}, Conf) when is_map(Source),
-                                                                               is_list(Conf) ->
+do_update({{?CMD_REPLACE, Type}, #{<<"enable">> := Enable} = Source}, Conf)
+  when is_map(Source), is_list(Conf), ?IS_BOOL(Enable) ->
     case create_dry_run(Type, Source)  of
```

Comment:
Maybe, we should try to improve the "dry run" part carefully. Currently, `dry_run` is called on rare occasions: when we _replace_ a single backend. It isn't called when we init the backends first time or add new backends.